WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -83,16 +83,16 @@ extension PluginDirectoryEntry: Codable {
         lastUpdated = try? container.decode(Date.self, forKey: .lastUpdated)
         rating = try container.decode(Int.self, forKey: .rating)
 
-        let icons = try? container.decodeIfPresent([String: String].self, forKey: .icons)
+        let icons = ((try? container.decodeIfPresent([String: String].self, forKey: .icons)) as [String : String]??)
         icon = icons??["2x"].flatMap(URL.init(string:))
 
         // If there's no hi-res version of the banner, the API returns `high: false`, instead of something more logical,
         // like an empty string or `null`, hence the dance below.
         let banners = try? container.nestedContainer(keyedBy: BannersKeys.self, forKey: .banners)
 
-        if let highRes = try? banners?.decodeIfPresent(String.self, forKey: .high) {
+        if let highRes = ((try? banners?.decodeIfPresent(String.self, forKey: .high)) as String??) {
```

Comment:
Maybe we could do:
`if let highRes = try banners?.decodeIfPresent(String.self, forKey: .high) {
            banner = URL(string: highRes)
        }`